addons/live-featured-project/addon.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/master/1/1.2.json",
```

Comment:
Hmm?

Reply:
This is not quite an issue. I just added it to add a check while implementing the settings properties.